Multisig/Logic/Models/PrivateKey/PrivateKey.swift

```diff
@@ -26,7 +26,7 @@ struct PrivateKey {
     }
 
     /// needed for PrivateKeyController.migrateLegacySigningKey()
-    var keyData: Data {
+    var legacyKeyData: Data {
```

Comment:
well, you know, sorry now to revert my previous comment - we need this data actually to show the private key in the export screen, I would not change it now.

Reply:
ok